pkg/nvcdi/lib.go

```diff
@@ -213,7 +228,7 @@ func (l *nvcdilib) resolveMode() (rmode string) {
 }
 
 // getCudaVersion returns the CUDA version of the current system.
-func (l *nvcdilib) getCudaVersion() (string, error) {
+func (l *nvcdilib) getCudaVersionNvml() (string, error) {
```

Comment:
nit: for this diff, let's keep the `getCudaVersion()` implementation here and then have the two implementations below this (for golang there is no need to have a function declared before its use).

Reply:
Didn't know that, thanks. Moved the two implementations below.